JSQMessagesViewController/Views/JSQMessagesTypingIndicatorFooterView.m

```diff
@@ -112,4 +117,34 @@ - (void)configureWithEllipsisColor:(UIColor *)ellipsisColor
     self.typingIndicatorImageView.image = [[UIImage jsq_defaultTypingIndicatorImage] jsq_imageMaskedWithColor:ellipsisColor];
 }
 
+- (void)configureWithAvatarImage:(UIImage *)avatarImage
+                         message:(NSString *)message
+                       textColor:(UIColor *)textColor
+                            font:(UIFont *)font;
+{
+    NSParameterAssert(avatarImage || message || textColor || font);
```

Comment:
What if you want to just show an avatar image, but no "X is typing…" message, or vice versa?